R/osrmfunctions.R

```diff
@@ -0,0 +1,206 @@
+#' Decode Google polyline compressed string
+#'
+#' @section Details:
+#' An implementation of the Google Maps Encoded Polyline Algorithm for decoding
+#' strings. Returns a dataframe if polyline is of length 1 and a list of
+#' dataframes otherwise.
+#'
+#' @param polyline A character string or vector of character strings containing
+#' the encoded polyline to be decoded.
+#' @param precision An integer indicating the number of decimals in the
+#' initial encoded coordinates. Default is 6 (for OSRM default).
+#' @export
+#' @examples \dontrun{
+#'  decode_google_lines("_p~iF~ps|U_ulLnnqC_mqNvxq`@@")
+#' }
+decode_google_lines <- function(polyline, precision=6) {
```

Comment:
Looks a little long - I suggest 'decode_gl': the 'Google lines' part is in the description